src/main/java/com/synopsys/integration/alert/component/scheduling/SchedulingUIConfig.java

```diff
@@ -43,11 +40,11 @@
     public static final String KEY_PURGE_DATA_FREQUENCY_DAYS = "purge.data.frequency";
     public static final String KEY_PURGE_DATA_NEXT_RUN = "purge.data.next.run";
 
-    @Override
-    public DescriptorMetadata generateDescriptorMetadata() {
-        return new DescriptorMetadata("Scheduling", "scheduling", SchedulingDescriptor.SCHEDULING_COMPONENT, DescriptorType.COMPONENT, ConfigContextEnum.GLOBAL, "clock-o", true, createFields());
+    public SchedulingUIConfig() {
+        super("Scheduling", "scheduling", "clock-o");
```

Comment:
Encapsulate lowercase scheduling and clock-o strings into static final strings in the descriptor class since they are the URL and font-awesome icon name.